packages/web-react/src/components/UNSTABLE_EmptyState/README.md

```diff
@@ -2,3 +2,79 @@
 
 ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
 Please use it with caution.
+
+EmptyState component should be included in the error pages or empty state pages as the carrier of information and links to elsewhere.
+
+EmptyState component is a composition of the following components:
+
+- [UNSTABLE_EmptyState](#unstable-emptystate)
+  - [UNSTABLE_EmptyStateSection](#unstable-emptystatesection)
+
+## UNSTABLE EmptyState
+
+The `UNSTABLE_EmptyState` component is a main container responsible for positioning the [UNSTABLE_EmptyStateSection](#unstable-emptystatesection) components or content.
+
+```jsx
+<UNSTABLE_EmptyState>
+  <!-- UNSTABLE_EmptyStateSection components or content go here -->
+</UNSTABLE_EmptyState>
+```
+
+### API
+
+The component is based on the [Stack][stack] component and accepts all its properties. For more information, see the [Stack component API][stack-api].
+
+On top of the API options, the components accept [additional attributes][readme-additional-attributes].
```

Comment:
Maybe these two sentences can be removed.

Reply:
removed